src/ica_api.c

```diff
@@ -1145,6 +1145,11 @@ unsigned int ica_rsa_mod_expo(ica_adapter_handle_t adapter_handle,
 	if (rsa_key->key_length * 8 > MAX_RSA_KEY_BITS)
 		return EPERM;
 
+#ifdef ICA_FIPS
+	if ((fips & ICA_FIPS_MODE) && (rsa_key->key_length * 8 < 2048))
```

Comment:
Brackets around `rsa_key->key_length * 8 < 2048` are not needed.
Same at the other 3 similar places further down.

Reply:
Done.